src/App.vue

```diff
@@ -30,7 +31,15 @@ export default {
       drinks: [],  
       showIngredients: true,
       ingredients: [] ,
-      filters: [] 
+      filters: [],
+      searchIngredients: ""  
+    }
+  },
+  computed: {
+    filteredIngredients: function(){
```

Comment:
Shorthand for this can just be

filteredIngredients() {

Reply:
I think I have updated it- not sure how to implement your changes automatically